server/shadow/shadow_client.c

```diff
@@ -817,85 +817,6 @@ int shadow_client_surface_update(rdpShadowClient* client, REGION16* region)
 	return 1;
 }
 
-int shadow_client_convert_alpha_pointer_data(BYTE* pixels, BOOL premultiplied,
```

Comment:
Sure, how about putting it in shadow_subsystem.c as a common function? Other subsystem implementations could use it.